torch/_export/__init__.py

```diff
@@ -120,6 +120,7 @@ def export(
     constraints: Optional[List[Constraint]] = None,
     *,
     _add_runtime_assertions=True,
+    _functionalize_runtime_assertions=False,
```

Comment:
The reason this flag exists is that we have a different signature with it, right? What would be the plan to turn it on?

Without this flag, how broken is the current state again? Will the assertion op still be DCE-d before future transforms can run? @tugsbayasgalan @angelayi can you remind me of the overall rollout plan here?

Reply:
So this flag is mostly for gating purpose before we could turn on this feature by default. But yeah I guess it's worth to discuss rollout plan as well. cc @tugsbayasgalan

> Without this flag, how broken is the current state again
My rough understanding is nothing is broken due to DCE since https://github.com/pytorch/pytorch/blob/618cc82e770cf758bc9cbcdfd4c61e2d73bdca25/torch/fx/node.py#L33-L41?